src/jqLite.js

```diff
@@ -587,6 +587,9 @@ forEach({
   },
 
   attr: function(element, name, value) {
+    if (element.nodeType === 2 || element.nodeType === 3 || element.nodeType === 8) {
```

Comment:
I've added the NODE_TYPE_ATTRIBUTE = 2 to src/Angular.js (also in the jshint comment at the top, to expose it as global). However, running grunt test still gives me an error that NODE_TYPE_ATTRIBUTE is not defined. I must be missing something.